### PR TITLE
Update block dragger

### PR DESCRIPTION
**Pull request by alschmiedt**
## The basics

- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Proposed Changes

- Changes private --> protected
- Changes package --> public
- Formats the document using clang
- Breaks up larger functions to make overriding easier
- Renames block dragger methods to make them more generic in anticipation of a common i_dragger interface.

### CODE REVIEW

File: core/block_dragger.js [hunk 1]
```diff
@@ -242,46 +262,73 @@ Blockly.BlockDragger.prototype.endBlockDrag = function(e, currentDragDeltaXY) {
...
+};
+
+/**
+ * Updates the necessary information to place a block at a certain location.
+ * @param {!Blockly.utils.Coordinate} delta The change in location from where
+ *     the block started the drag to where it ended the drag.
+ * @protected
+ */
+Blockly.BlockDragger.prototype.updateBlockAfterMove_ = function(delta) {
+  // These are expensive and don't need to be done if we're deleting.
```
Comment (maribethb) [inline core/block_dragger.js:279]: nit: this comment makes more sense where it was before imo (explaining why we are only calling this when not deleting)

**Review by maribethb — APPROVED** [review]
We are currently discussing offline how to handle the name changes... since this isn't going into develop right now I'm fine with however you choose to handle it, we should just make sure to revisit this conversation before merging to develop

I like the new functions! Makes it easier to read as well as override now

File: core/block_dragger.js (hunk 1 above)
Comment (alschmiedt) [reply core/block_dragger.js]: Updated.

**Comment by alschmiedt** [comment]
I'm going to leave the names for right now so that we can get this in, but we can revisit next week.

**Pull request merged**